sql-cli/tests/test___main__.py

```diff
@@ -47,3 +50,33 @@ def test_generate(root_directory, dags_directory):
 def test_validate():
     result = runner.invoke(app, ["validate"])
     assert result.exit_code == 0
+
+
+def test_init_with_directory():
+    with tempfile.TemporaryDirectory() as dir_name:
+        result = runner.invoke(app, ["init", dir_name])
+        assert result.exit_code == 0
+        expected_msg = f"Initialized an Astro SQL project at {dir_name}"
+        assert expected_msg in get_stdout(result)
+        assert list_dir(dir_name)
+
+
+def test_init_with_custom_airflow_config():
+    with tempfile.TemporaryDirectory() as dir_name:
```

Comment:
Can we not use `runner.isolated_filesystem()` here as well instead of `tempfile.TemporaryDirectory()`?

Reply:
The isolated filesystem changes the current directory to be the temporary one being created. I didn't want this behaviour in this test - I wanted the current directory to be pointing to a different folder.